keyboards/geekboards/tester/readme.md

```diff
@@ -0,0 +1,14 @@
+Geekboards 8-keys macropad
+=====
+
+Keyboard Maintainer: Geekboards
```

Comment:
The text here should be a link to the maintainer's GitHub profile. There should also be two spaces at the end of the line.

*e.g.*:

```suggestion
Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)  
```